session 1 changes for 2025

luisesanmartin opened the pull request:
Summary:
- updated dates and links
- added a screenshot showing how importing data manually implicitly applies two functions in RStudio

luisesanmartin commented:
@mariarrt94 I ad an issue I couldn't solve when rendering the slides into HTML and PDF. Can you pull this branch to your computer and upload the HTML or PDF so Mahrukh can review them? thanks a lot!

luisesanmartin commented:
@mariarrt94 if it also fails for you, try rendering the slides in your personal computer

mariarrt94 commented:
@luisesanmartin I added the presentations. There is an issue with the shiny app we are using to show the code it keeps opening and opening new pages of the app. For my slides I removed the app and I will show it live.

luisesanmartin commented on Presentations/01-intro-to-R.Rmd:
```diff
@@ -815,7 +816,7 @@ help(paste)
 str_vec <- c("R", "Python", "SAS", "Excel", "Stata")
 str_scalar <- "can be an option to"       # creating str_scalar
-paste(str_vec[1], str_scalar, str_vec[5]) # using paste()
+paste(str_vec[1], str_scalar, str_vec[4]) # using paste()
```
added

luisesanmartin commented on Presentations/01-intro-to-R.Rmd:
```diff
@@ -303,7 +306,7 @@ knitr::include_graphics("img/View.png")
 # Data in R
 
-Alternatively we can print the first 6 obs. with `head()`:
+To explore data, we can print the first 6 obs. with `head()`:
```
added `tail()` but not `as_tibble() %>% print()` as pipes are introduced in session 2 and explaining tibbles might confuse them since it's the first session.

luisesanmartin commented on Presentations/01-intro-to-R.Rmd:
added in the last slide

luisesanmartin commented:
> @luisesanmartin I added the presentations. There is an issue with the shiny app we are using to show the code it keeps opening and opening new pages of the app. For my slides I removed the app and I will show it live.

yes, I'm really puzzled as to why sometimes it works and sometimes it doesn't. I wanted try `webr` as well as an option, but didn't have time to. I'll check if it's working tomorrow morning, otherwise I'll show and explain some of the code live in the session
